graphql_compiler/tests/conftest.py

```diff
@@ -30,25 +33,45 @@ def init_integration_orientdb_client():
     return _init_orientdb_client(load_schema, generate_orient_integration_data)
 
 
+# retry is a decorator that attempts to call the decorated function repeatedly, and it takes a
+# parameter "call". Call is the function being decorated, so we don't need to specify that in
+# the parameters that the retry decorator takes, so we can disable pylint's warning about
```

Comment:
Can you share the error pylint raises here? This comment effectively states that pylint has buggy handling of decorators, which is definitely possible but bears verifying before we silence the error.

Also, `tries` is a positional argument, not a kwarg -- so let's pass it positionally.

Reply:
in PR 410 https://github.com/kensho-technologies/graphql-compiler/pull/410#discussion_r304635008